dockers/docker-ptf/Dockerfile.j2

```diff
@@ -51,6 +51,7 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         hping3              \
         curl                \
         tcpdump             \
+        tmux                \
```

Comment:
What is the benefit provided by including tmux?

Reply:
When I run tests overnight I need something inside of ptf container to be working in background.